src/components/guess.js

```diff
@@ -4,10 +4,11 @@ import { connect } from 'react-redux';
 class Guess extends Component {
   render() {
     const guess = this.props.guess;
+    console.log(this.props);
     if (guess.get('status') === "correct") {
       return (
           <div className="correct guess">
-            Correct!
+            Correct! Their Full Name is {this.props.person.first_name} {this.props.person.last_name}
```

Comment:
If you want to stay in the [Immutable.js](https://facebook.github.io/immutable-js/) world, you could say `this.props.person.get('first_name')` - or, better, extract a `person` variable next to the `guess`, and then just `person.get('first_name')`. Then you wouldn't need to `.toJS()` below.